components/private/Snapshot/MDCSnapshotTestsStrings.h

```diff
@@ -14,30 +14,30 @@
 
 #pragma mark - Latin text
 
-static NSString *const MDCTextFieldSnapshotTestsPlaceholderShortTextLatin = @"P text";
-static NSString *const MDCTextFieldSnapshotTestsPlaceholderLongTextLatin =
+static NSString *const MDCSnapshotTestsPlaceholderShortTextLatin = @"P text";
```

Comment:
These English strings don't really make sense outside of Text Fields usage. These strings (at least the English ones) are unique to their placement within the text field. For example, "P text" represents Placeholder text and "I text" is Input text.  The Arabic strings are just short/long Lorem Ipsum generated at http://generator.lorem-ipsum.info/_arabic . 

I would suggest the following:
*  Copy all the strings into `private/Snapshot` but regenerate the Latin text from Lorem Ipsum.
* In TextFields snapshot strings, keep the current Latin values (they would be unique to Text Fields) and use the `private/Snapshot` strings for Arabic.
* Other components can use the more general Lorem Ipsum values defined in `private/Snapshot` when appropriate and can generate custom strings as needed.

What do you think?

Reply:
I can agree with that, I was going to write some snapshot test and needed strings so instead of making a new file or declaring new variables I tried to just move these. I think they should all be lorem-ipsum.